src/services/token.service.js

```diff
@@ -0,0 +1,27 @@
+import { Token } from "../models/token.js";
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId }});
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+    return;
+  }
+
+  token.refreshToken = newToken;
+  await token.save();
+}
+
+function getByToken(refreshToken) {
+  return Token.findOne({ where: { refreshToken } });
+}
+
+function remove(userId) {
+  return Token.destroy({ where: { userId } })
+}
+
+export const toeknService = {
```

Comment:
There is a typo in the export statement. It should be `tokenService` instead of `toeknService`.